scripts/end-to-end/verify-testcases.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+#
+# - SolidityEndToEndTest.trace was created with soltest with the following command on
+# develop @ commit 9318dae42cfd1a0f048665ec1eafae730bf7cb6b:
+#     ./soltest --log_level=test_suite -t SolidityEndToEndTest/ -- --no-smt
+#         --evmonepath /Users/alex/evmone/lib/libevmone.dylib --show-messages > SolidityEndToEndTest.trace
```

Comment:
```suggestion
#     ./soltest --log_level=test_suite -t SolidityEndToEndTest/ -- --show-messages > SolidityEndToEndTest.trace
```

Reply:
If you disable color output, you don't get the annoying ANSI escapes in the trace.